src/toolbox.ts

```diff
@@ -1408,46 +1507,64 @@ export class AnnotationResizeItem extends ToolboxItem {
                     subtask.annotations.access[annotation_id].line_size += size;
                     break;
                 case "-":
-                    // Check to make sure annotation line size won't go 0 or negative. 
+                    // Check to make sure annotation line size won't go 0 or negative.
                     // If it would, set it equal to a small positive number
-                    if (subtask.annotations.access[annotation_id].line_size - size <= 0.01) {
-                        subtask.annotations.access[annotation_id].line_size = 0.01
+                    if (
+                        subtask.annotations.access[annotation_id].line_size -
+                            size <=
+                        0.01
+                    ) {
+                        subtask.annotations.access[annotation_id].line_size =
+                            0.01;
                     } else {
-                        subtask.annotations.access[annotation_id].line_size -= size;
+                        subtask.annotations.access[annotation_id].line_size -=
+                            size;
                     }
                     break;
                 default:
-                    throw Error("Invalid Operation given to loop_through_annotations")
+                    throw Error(
+                        "Invalid Operation given to loop_through_annotations",
+                    );
             }
         }
 
         if (subtask.annotations.ordering.length > 0) {
-            this.set_size_cookie(subtask.annotations.access[subtask.annotations.ordering[0]].line_size, subtask);
+            this.set_size_cookie(
+                subtask.annotations.access[subtask.annotations.ordering[0]]
+                    .line_size,
+                subtask,
+            );
         }
     }
 
     //Loop through all subtasks and apply a size to them all
     public update_all_subtask_annotation_size(ulabel, size) {
-        for (let subtask in ulabel.subtasks) {
-            this.update_annotation_size(ulabel, ulabel.subtasks[subtask], size)
+        for (const subtask in ulabel.subtasks) {
+            this.update_annotation_size(ulabel, ulabel.subtasks[subtask], size);
         }
     }
 
     private set_size_cookie(cookie_value, subtask) {
-        let d = new Date();
-        d.setTime(d.getTime() + (10000 * 24 * 60 * 60 * 1000));
+        const d = new Date();
+        d.setTime(d.getTime() + 10000 * 24 * 60 * 60 * 1000);
 
-        let subtask_name = subtask.display_name.replaceLowerConcat(" ", "_");
+        const subtask_name = subtask.display_name.replaceLowerConcat(" ", "_");
 
-        document.cookie = subtask_name + "_size=" + cookie_value + ";" + d.toUTCString() + ";path=/";
+        document.cookie =
+            subtask_name +
+            "_size=" +
+            cookie_value +
+            ";" +
+            d.toUTCString() +
+            ";path=/";
```

Comment:
Not a massive fan of wraps like this. Could we just use string interp instead?
```typescript
document.cookie = `${subtask_name}_size=${cookie_value};${d.toUTCString()};path=/`;
```
or even break it into a couple intermediate statements.

Reply:
Or a generic cookie utility - I think there's only this and the night mode cookie though.